src/lib/contract.js

```diff
@@ -92,3 +95,45 @@ export const ERC20Funcs = [
         outputs:[{ name: '', type: 'uint256'}]
     }),
 ]
+
+
+export const joinTheReg =  Immutable.fromJS({
+  name: 'register',
+  inputs:[{name:"_tAddr", type : "address"},
+          {name:"_tSale", type: "address"},
+          {name:"_tName", type: "string"},
+          {name:"_tSymbol", type:"string"},
+          {name:"_tDecimal",type:"uint"},
+          {name:"_tType",type:"string"},
+          {name:"_tIcon",type:"string"},
+          {name:"_tURL",type:"string"},
+          {name:"_tBlerb",type:"string"}]
+});
+
+export const regCount =  Immutable.fromJS({
+        name: "Count",
+        inputs:[],
+        outputs:[{ name: '', type: 'uint'}]
+      });
+
```

Comment:
ok, so proper usage is to call `regCount`, then iterate through the indices with `getArray`?